README.md

```diff
@@ -129,12 +129,25 @@ You can also specify build metadata in a version tag. If the tag is on the curre
 
 ### Can I use the version calculated by MinVer for other purposes?
 
-Yes! MinVer sets both the `Version` and `PackageVersion` MSBuild properties. Use them in a target which runs after MinVer. E.g.
+Yes! MinVer sets the `Version`, `PackageVersion`, `AssemblyVersion`, `MinVerMajor`, `MinVerMinor`, and `MinVerPatch` MSBuild properties. You can use them, or override their values, in a target which runs after MinVer.

```

Comment:
Should this be updated to reflect my change of also needing to set `FileVersion`? I suppose maybe not since it's only being set to maintain the value it would have if `AssemblyVersion` wasn't set.

Reply:
I'll include `FileVersion` too.